app/src/main/java/com/android/streetworkapp/ui/map/MapUi.kt

```diff
@@ -52,14 +57,18 @@ fun MapScreen(parkLocationViewModel: ParkLocationViewModel, navigationActions: N
 
         // Display the Google Map
         GoogleMap(
+            onMapLoaded = callbackOnMapLoaded,
```

Comment:
Question: what is "callbackOnMapLoaded" used for?

Reply:
for the e2e test, it decreases the counter of an idlingRessource that deblocks the test and makes it resume. Otherwise you can't click on Markers as they aren't loaded